spec/ParseUser.spec.js

```diff
@@ -4031,4 +4031,30 @@ describe('Security Advisory GHSA-8w3j-g983-8jh5', function () {
     const user = await query.get('1234ABCDEF', { useMasterKey: true });
     expect(user.get('authData')).toEqual({ custom: { id: 'linkedID' } });
   });
+
+  describe('issue #6641', () => {
+    fit('succes to login when masterkey and instalationId is provided and when password is not provided', done => {
```

Comment:
Please use the async/await syntax, instead of chained promises; and keep the description concise, e.g.:

```
it('sign up with 3rd party auth via cloud function returning session token', async () => {
  const user = ...;
  await user.save(...);
  ...
});

```